### ISSUE TITLE
Add secure tags (resource_manager_tags) on compute_instance module

### ISSUE THREAD

**Issue by Samir-Cit** [terraform-google-modules/terraform-google-vm#343]
### TL;DR

The `google_compute_instance_from_template` resource supports adding a [resource_manager_tags](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_instance#resource_manager_tags), but it's not exposed as a variable in this module.

### Terraform Resources

[terraform-google-vm/modules/compute_instance](https://github.com/terraform-google-modules/terraform-google-vm/tree/master/modules/compute_instance)

[google_compute_instance_from_template](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_instance_from_template)
[google_compute_instance](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_instance)

### Detailed design

```terraform
variable "resource_manager_tags" {
  type        = map(string)
  default     = {}
}

resource "google_compute_instance_from_template" "compute_instance" {
  ...
  params {
    resource_manager_tags = var.resource_manager_tags
  }
  ...
}
```

### Additional information

I would like to set security tags to use in a VM to bind it to a Firewall Policy Rule.
